qt/aqt/forms/preferences.ui

```diff
@@ -798,6 +798,22 @@
             </property>
            </widget>
           </item>
+          <item>
+           <widget class="QPushButton" name="syncLogin">
+            <property name="sizePolicy">
+             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
+              <horstretch>0</horstretch>
+              <verstretch>0</verstretch>
+             </sizepolicy>
+            </property>
+            <property name="text">
+             <string notr="true">LOGIN</string>
```

Comment:
Instead of setting the translation in the Python code, you can name it here instead, eg sync_log_in_button.